### PR TITLE
Add slackBusPMaxMismatch parameter

### PR DESCRIPTION
**Pull request by obrix**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

Add slackBusPMaxMismatch  parameter with a default value of 0.01

**What is the new behavior (if this is a feature change)?**

A new parameter is available in .yml files.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

### CODE REVIEW

**Review by annetill — CHANGES_REQUESTED** [review]
How does it work? With your PR, you are able to change a parameter, but where is this parameter used in the code? Then, you have to reach 90% of test coverage in OLF.

**Comment by obrix** [comment]
@annetill This is a new attribute in OpenLoadFlowParameter class, it is used the same way as other parameters like throwsExceptionInCaseOfSlackDistributionFailure.
I just plugged it into DistributedSlackOuterLoop class constructor which is the reference you gave to me. If you think it should also be passed elsewhere, let me know. Coverage is fixed now that the parameter is used.

File: src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java [hunk 1]
```diff
@@ -45,6 +46,8 @@
 
     private boolean addRatioToLinesWithDifferentNominalVoltageAtBothEnds = ADD_RATIO_TO_LINES_WITH_DIFFERENT_NOMINAL_VOLTAGE_AT_BOTH_ENDS_DEFAULT_VALUE;
 
+    private double slackBusPMaxMismatch = SLACK_BUS_P_MAX_MISMATCH_DEFAULT_VALUE / PerUnit.SB;
```
Comment (geofjamg) [inline src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java:49]: You should keep it in MW in the parameters

File: src/main/java/com/powsybl/openloadflow/ac/DistributedSlackOuterLoop.java [hunk 2]
```diff
@@ -49,7 +47,7 @@ public String getType() {
     @Override
     public OuterLoopStatus check(OuterLoopContext context, Reporter reporter) {
         double slackBusActivePowerMismatch = context.getLastNewtonRaphsonResult().getSlackBusActivePowerMismatch();
-        if (Math.abs(slackBusActivePowerMismatch) > SLACK_BUS_P_MAX_MISMATCH) {
+        if (Math.abs(slackBusActivePowerMismatch) > slackBusPMaxMismatch) {
```
Comment (geofjamg) [inline src/main/java/com/powsybl/openloadflow/ac/DistributedSlackOuterLoop.java:50]: This is here that you should convert to PU

**Review by geofjamg — CHANGES_REQUESTED** [review]

File: src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java [hunk 3]
```diff
@@ -116,6 +119,15 @@ public OpenLoadFlowParameters setPlausibleActivePowerLimit(double plausibleActiv
         return this;
     }
 
+    public double getSlackBusPMaxMismatch() {
+        return slackBusPMaxMismatch;
+    }
+
+    public OpenLoadFlowParameters setSlackBusPMaxMismatch(double pSlackBusPMaxMismatch) {
+        this.slackBusPMaxMismatch = pSlackBusPMaxMismatch / PerUnit.SB;
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java:127]: I get it, so I remove the per unit here.

**Review by geofjamg — APPROVED** [review]

**Pull request merged**